src/utils/html.rs

```diff
@@ -42,118 +43,127 @@ pub(crate) fn rewrite_rustdoc_html_stream<R>(
 where
     R: AsyncRead + Unpin + 'static,
 {
+    let stream_span = tracing::info_span!("rewrite_rustdoc_html_stream");
+
     stream!({
         let (input_sender, input_receiver) = std::sync::mpsc::channel::<Option<Vec<u8>>>();
         let (result_sender, mut result_receiver) = tokio::sync::mpsc::unbounded_channel::<Bytes>();
 
-        let join_handle: JoinHandle<anyhow::Result<_>> = tokio::spawn(async move {
```

Comment:
this _looks_ odd, but it's only whitespace changes. 

Only thing I did was to attach `.instrument(span)` to the end of the async block